content/en/blog/2023-08-28-announcing-flux-v2.1.0/index.md

```diff
@@ -0,0 +1,274 @@
+---
+author: somtochiama
+date: 2023-08-28 00:00:00+00:00
+title: Announcing Flux 2.1 GA
+description: "Flux v2.1.0: New monitoring setup with support for custom metrics, performance improvements and much more!"
+url: /blog/2023/08/flux-v2.1.0/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+## New releases
+
+We are happy to announce the latest GA releases for Flux and Flagger.
+
+### Flux v2.1.0
+
+This new release comes with lots of new features,
+fixes, restructured documentation and performance improvements.
+Everyone is encouraged to upgrade for the best experience.
+
+The [Flux APIs](https://github.com/fluxcd/flux2/releases/tag/v2.1.0#api-changes)
+were extended with new opt-in features in a backwards-compatible manner.
+
+The Flux Git capabilities have been improved with support for
+Git push options, Git refspec, Gerrit, HTTP/S and SOCKS5 proxies.
+
+In case you missed it, Flux reached General Availability in June.
+You can read the announcement
+[here](https://fluxcd.io/blog/2023/07/flux-ga/).
+
+You can now check the end-of-life(EOL) dates and support information for 
+different Flux versions at https://endoflife.date/flux.
+
+#### Features
```

Comment:
this section should also include support for Git tag verification in SC and support for push options, push refspec in IAC

Reply:
I don't think we need to put in all new features in the blog post, just highlights. I will add a note for just the IAC feature.

Reply:
okay, then it'd be better to mention the tag verification feature over the IAC feature

Reply:
I agree with @aryan9600, Git tag verification should be listed in features.